### PR TITLE
app: Run cargo fmt

### PR DESCRIPTION
**Pull request by alistair23**
Run Rust's automated style checker with the `cargo fmt` command.

Signed-off-by: Alistair Francis <alistair@alistair23.me>

### CODE REVIEW

**Comment by xou816** [comment]
Hi :) 
Thank you for your contribution. I haven't had the time to play with rustfmt but this does look promising! 
I am not sure I like the defaults (widths in particular), and I'd rather take the time to setup at least a git hook or a basic CI to enforce the style before merging style changes. I hope you understand!

**Comment by alistair23** [comment]
Done!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# This workflow contains all tock-ci, seperated into jobs
```
Comment (xou816) [inline .github/workflows/ci.yml:1]: leftover comment? :p

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          default: true
+          components: rustfmt, clippy
```
Comment (xou816) [inline .github/workflows/ci.yml:22]: clippy not needed just yet but okay

**Review by xou816 — COMMENTED** [review]
Thanks! 👍 
I had started another workflow for building+testing, I'll look into merging them with this workflow possibly (although it might be good to run "light" checks such as cargo fmt on all push events, and limit build/test to master)

File: .github/workflows/ci.yml (hunk 2 above)
Comment (alistair23) [reply .github/workflows/ci.yml:18]: Yep, it's also something worth running eventually so I left it in.

**Pull request merged**